### PR TITLE
alert rules: fix alert items

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)

Fixed some alert items in "alert-rules.md".
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7275
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by rleungx — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

File: alert-rules.md [hunk 1]
```diff
@@ -243,32 +243,32 @@ This section gives the alert rules for the PD component.
 
 ### Warning-level alerts
 
-#### `PD_cluster_lost_connect_tikv_nums`
+#### `PD_cluster_lost_connect_store_nums`
 
 * Alert rule:
 
-    `sum(pd_cluster_status{type="store_disconnected_count"}) > 0`
+`(sum(pd_cluster_status{type="store_disconnected_count"}) by (instance) > 0) and (sum(etcd_server_is_leader) by (instance) > 0)`
```
Comment (lilin90) [inline alert-rules.md:250]: Please comply with the space indentation rules in a list, and keep the style of similar items consistent in this file.
```suggestion
    `(sum(pd_cluster_status{type="store_disconnected_count"}) by (instance) > 0) and (sum(etcd_server_is_leader) by (instance) > 0)`
```

File: alert-rules.md [hunk 2]
```diff
@@ -314,7 +314,7 @@ This section gives the alert rules for the PD component.
 
 * Alert rule:
 
-    `sum(pd_regions_status{type="down_peer_region_count"}) > 0`
+    `(sum(pd_regions_status{type="down-peer-region-count"}) by (instance) > 0) and (sum(etcd_server_is_leader) by (instance) > 0)`
```
Comment (en-jin19) [inline alert-rules.md:317]:
```suggestion
    `(sum(pd_regions_status{type="down-peer-region-count"}) by (instance)  > 0) and (sum(etcd_server_is_leader) by (instance) > 0)`
```

File: alert-rules.md (hunk 1 above)
Comment (en-jin19) [reply alert-rules.md]: > Please comply with the space indentation rules in a list, and keep the style of similar items consistent in this file.

Have fixed the problem, thank you!

File: alert-rules.md [hunk 3]
```diff
@@ -230,7 +230,7 @@ This section gives the alert rules for the PD component.
 
 * Alert rule:
 
-    `sum(pd_regions_status{type="miss_peer_region_count"}) > 100`
+    `(sum(pd_regions_status{type="miss_peer_region_count"}) by (instance) > 100) and (sum(etcd_server_is_leader) by (instance) > 0)`
```
Comment (en-jin19) [inline alert-rules.md:233]:
```suggestion
    `(sum(pd_regions_status{type="miss_peer_region_count"}) by (instance)  > 100) and (sum(etcd_server_is_leader) by (instance) > 0)`
```

**Pull request merged**